### ISSUE TITLE
Update to NodeJS v16

### ISSUE THREAD

**Issue by wincent** [wincent/masochist#179]
Currently using v14.17.1 on the server, but current LTS version is v16.13.1.

Wouldn't be too hard to update because I am installing and building from source already anyway (via Ansible).

Nothing really pressing motivating this; just wanting to stay relatively up-to-date to avoid later headaches.

**Comment by wincent** [comment]
Just tried going to 18.17.0, but had to back down to 16.21.0 — citing my commit from ansible-configs:

---

#### chore: downgrade NodeJS from v18.17.0 to v16.20.1

Same error as others reported here:

- https://stackoverflow.com/questions/72022527/glibc-2-27-not-found-while-installing-node-on-amazon-ec2-instance

Amazon Linux 2 only has glibc 2.26:

    $ ldd --version
    ldd (GNU libc) 2.26
    Copyright (C) 2017 Free Software Foundation, Inc.
    This is free software; see the source for copying conditions.  There is NO
    warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
    Written by Roland McGrath and Ulrich Drepper.

So trying to run NodeJS v18 will die:

    $ /opt/node/bin/node --version
    /opt/node/bin/node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /opt/node/bin/node)
    /opt/node/bin/node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /opt/node/bin/node)

Updating glibc isn't viable, so will have to wait until moving to Amazon
Linux 2023:

- https://github.com/wincent/masochist/issues/175

Amazon Linux 2023 comes with NodeJS v18 anyway (18.12.1 at the time of
writing):

- https://github.com/amazonlinux/amazon-linux-2023/blob/main/Release-Notes-Amazon-Linux-2023.0.20230614.md

---

So, retitled this to "Update to NodeJS v16" and will close it, then create another issue for v18, marking it as blocked by the Amazon Linux 2023 ticket.
